rest-model/src/main/java/org/jboss/pnc/rest/restmodel/BuildExecutionConfigurationRest.java

```diff
@@ -95,6 +116,20 @@ public BuildExecutionConfiguration toBuildExecutionConfiguration() {
         );
     }
 
+    public BuildConfigurationAudited toBuildConfigurationAudited() {
+        BuildConfigurationAudited buildConfigAudited = new BuildConfigurationAudited();
```

Comment:
I don't see it as a conversion as the two entities are not representing the same data. At the end the two entities might live on two different deployments (coordinator & executor).
Can't you work directly with this entity?